README.md

```diff
@@ -1,17 +1,267 @@
+git pull --rebase --prune
+```
+
+With `hit` you can just:
+
+```shell
+hit hop
+```
+
+### hit issue
+
+Now you need to decide which issue you should work at. You can use the `hit issue` command to see the full list of all open issues. After choosing the number of the issue, let's say 42, call `hit issue 42` to see the details of that issue.
+
+### hit new
+
+Start your work in a new branch. According to our workflow, branch names should have the following form:
+
+```
+<user_login>/<issue_number>-<short_issue_description>
+```
+
+With `git` you can create a branch using the following command:
+
+```shell
+git checkout -b my-login/42-short-desc
+```
+
+`hit` allows to accomplish this task easier:
+
+```shell
+hit new 42
+```
+
+It uses the issue title to generate a short description.
+
+### hit commit
+
+After finishing your work on that issue, you need to commit your changes. With `git` you would do the following:
+
+```shell
+git add .
+git commit -m "[#42] Implement my feature
+
+Resolves #42"
+```
+
+With `hit` you need only to specify the text of the commit to get the same result:
+
+```shell
+hit commit "Implement my feature"
+```
+
+Note that you don't need to keep in mind the current issue number. However, if you want to refresh the context about the issue, use the `hit current` command.
+
+### hit push
+
+After committing your changes locally, you need to push them to the remote repository. It's usually a good practice only to push the current branch.
+
+`git` command for this is a little bit verbose:
+
+```shell
+git push -u origin my-login/42-short-desc
+```
+
+`hit` allows you to save several keystrokes:
+
+```shell
+hit push
+```
+
+### hit sync
+
+After opening the pull request, some of the reviewers suggested changes that you applied as commits to the remote branch via GitHub interface. Now you need to sync your local branch with the remote one.
+
+With `git` you can do the following:
+
+```shell
+git pull --rebase origin my-login/42-short-desc
+```
+
+However, with `hit` you can just:
+
+```shell
+hit sync
+```
+
+### hit fresh
+
+While you were waiting for the second review round, another pull request was merged to the `master` branch. Now you need to apply the new `master` changes to your local branch.
+
+With `git` you can do the following:
+
+```shell
+git fetch origin master
+git rebase origin/master
+```
+
+Again, with `hit` you can do better:
+
+```shell
+hit fresh
+```
+
+### hit fix
+
+Now you need to make changes to your work locally according to the code review and push them to the remote repository.
+
+`git` requires from you to do several steps to accomplish this simple task:
+
+```shell
+git add .
+git commit -m "Fix after review"
+git push origin my-login/42-short-desc
+```
+
+`hit` helps you with this as well:
+
+```shell
+hit fix
+```
+
+### hit amend
+
+Oops, you've just realised that you have made a typo in your work! Now you want to fix this problem but without creating new unnecessary commit.
```

Comment:
Maybe we should mention here that the following commands should be run after fixing the typo. I had to read this a couple of times before I got it. Just a suggestion